.circleci/config.yml

```diff
@@ -52,10 +52,10 @@ jobs:
     <<: *defaults
     parameters:
       docker-compose-filename:
-        default: docker-compose.circleci.yml
+        default: docker-compose.yml
```

Comment:
Circleci now uses the same docker-compose file as the local environment (so there's no need to manage to separate files).